org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -186,14 +213,30 @@ public void importToWorkspace(IProgressMonitor monitor) throws CoreException {
 				JavaLanguageServerPlugin.logException("Failed to update digest for gradle build file", e);
 			}
 		});
+		for (IProject gradleProject : ProjectUtils.getGradleProjects()) {
+			gradleProject.deleteMarkers(COMPATIBILITY_MARKER_ID, true, IResource.DEPTH_ZERO);
+		}
+		for (IStatus status : compatibilityStatus.getChildren()) {
+			// only report first compatibility issue
+			GradleCompatibilityStatus gradleStatus = ((GradleCompatibilityStatus) status);
+			for (IProject gradleProject : ProjectUtils.getGradleProjects()) {
+				if (JDTUtils.getFileURI(gradleProject).equals(URI.create(gradleStatus.getProjectUri()).toString())) {
```

Comment:
I didn't see the error market emitted and tracked it down to :

LS : `file:///home/rgrunber/sample-projects/gradle-simple`
RS : `file:///home/rgrunber/sample-projects/gradle-simple/`

Reply:
I can reproduce this in mac. And it might because in unix systems the `URI` created from `java.nio.Path` and `JDTUtils.getFileURI(IResource)` are not the same (with an extra tailing slash), https://github.com/eclipse/eclipse.jdt.ls/pull/1965/commits/24c0ee9dc1a64d4e6c89d19c7a49124aa216b348 use `URIUtil.sameURI()` to fix this.